modules/auth-on-llama-stack.adoc

```diff
@@ -0,0 +1,388 @@
+:_module-type: PROCEDURE
+
+[id="auth-on-llama-stack_{context}"]
+= Configuring Llama Stack with OAuth Authentication
+
+You can configure Llama Stack to enable Role-Based Access Control (RBAC) for model access using OAuth authentication on {productname-short}. The following example shows how to configure Llama Stack so that a vLLM model can be accessed by all authenticated users, while an OpenAI model is restricted to specific users. This is example uses keycloak to issue and validate tokens.
+
+Before starting, you must already have KeyCloak set up with the following parameters:
```

Comment:
I think Keycloak is how we mention it? I see it formatted differently in this PR so might be worth standardizing. 

Small rec: 
"Before you begin, you must already have Keycloak set up with the following parameters:"